infer/src/backend/InferPrint.re

```diff
@@ -308,7 +308,8 @@ let should_report (issue_kind: Exceptions.err_kind) issue_type error_desc eclass
       switch Config.analyzer {
       | Checkers
       | Eradicate
-      | Tracing => true
+      | Tracing
+      | Bo => true
```

Comment:
For consistency with the other (non-abbreviated) names, we probably want to call this something like `BufferOverrun` or `Buffers` or `Overrun`.